build/lpc55pins/src/lib.rs

```diff
@@ -155,6 +157,21 @@ pub fn codegen(pins: Vec<PinConfig>) -> Result<()> {
         writeln!(&mut file, "{}", p.to_token_stream())?;
         writeln!(&mut file, ");")?;
 
+        // Output pins can specify their value, which is set before configuring
+        // their output mode (to avoid glitching).
+        if let Some(v) = p.value {
+            assert!(
+                matches!(p.direction, Some(Direction::Output)),
+                "can only set value for output pins"
+            );
+            writeln!(&mut file, "iocon.set_val(")?;
+            writeln!(&mut file, "{}", p.pin.to_token_stream())?;
```

Comment:
**question:** Does `to_token_stream` inject a trailing comma? There's no comma between the first and second parameters to `set_val` that I can see.